src/api/tests/feature_tests/features/searchCpmProduct.failure.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Search CPM Products - failures scenarios
```

Comment:
Will there be any failure scenarios for missing parameters or extra parameters?

Reply:
Any other path params would cause a 404
I can see a future where we might want to pass in query params, however currently we don't parse any incoming query params so they would just be ignored. And it wouldn't fail. 
Same for body

Reply:
Ah yeah, I was getting confused between query and path params